crypto/des/qud_cksm.c

```diff
@@ -19,6 +19,9 @@
  * use.
  */
 #include "internal/deprecated.h"
+#include <stdint.h>  
+
+typedef uint32_t DES_LONG;
```

Comment:
I'm not sure about the wisdom of adding this, since `DES_LONG` already has a typedef in `include/openssl/des.h`